docs/en/serverless/quickstarts/k8s-logs-metrics.mdx

```diff
@@ -0,0 +1,45 @@
+---
+slug: /serverless/observability/quickstarts/k8s-logs-metrics
+title: Kubernetes logs and metrics
```

Comment:
I've changed the title of the other QS. Assuming folks like the change, we should use parallel titles, so this should be something like "Monitor Kubernetes."